dg/config/cluster_credentials_manager.py

```diff
@@ -204,6 +207,36 @@ def get_current_cluster(self) -> Union[AbstractCluster, None]:
 
         return cluster
 
+    def get_current_credentials(self) -> ContextData:
+        """Returns user and current cluster credentials
+
+        Returns
+        -------
+        ContextData
+            user and current cluster credentials
+        """
+
+        dg_credentials_file_path = (
+            data_gate_configuration_manager.get_dg_credentials_file_path()
+        )
+        result: ContextData
+
+        if dg_credentials_file_path.exists() and (
+            dg_credentials_file_path.stat().st_size != 0
+        ):
+            with open(dg_credentials_file_path) as json_file:
+                result = json.load(json_file)
+        else:
+            result = {}
+
+        current_cluster = self.get_current_cluster()
+
+        if current_cluster is not None:
+            result.update(current_cluster.get_cluster_data())
+            result["server"] = current_cluster.get_server()
+
+        return result
```

Comment:
Same as the reasoning above. What happens in the empty result case? I'd opt for throwing an exception if no current credentials could be obtained.

Reply:
`get_current_credentials()` always returns a dictionary. An empty dictionary is a valid scenario (no stored credentials, no selected cluster). In this case, the required parameters must be explicitly passed to the respective command. Therefore, no exception must be thrown.